server/middleware/getUnallocatedCasesCount.ts

```diff
@@ -0,0 +1,30 @@
+import { RequestHandler } from 'express'
+import logger from '../../logger'
+import UserPreferenceService from '../services/userPreferenceService'
+import AllocationsService from '../services/allocationsService'
+
+export default function getUnallocatedCasesCount(
+  userPreferenceService: UserPreferenceService,
+  allocationsService: AllocationsService
+): RequestHandler {
+  return async (req, res, next) => {
+    try {
+      if (res.locals.user) {
+        const { token, username } = res.locals.user
+        const { items: teamSelection } = await userPreferenceService.getTeamsUserPreference(token, username)
+        // backwards compatibility for current journey, once multiple teams released this should be removed
```

Comment:
Am I right in thinking that this will cause the badge to appear during the team selection journey if there are any cases in Wrexham?

Reply:
yeah, this ticket will be blocked until we integrate the team selection journey into the main journey (or we create a separate ticket for removing the backwards compatibility after integrating)